app/views/ReportView/components/RapidSummary/index.tsx

```diff
@@ -57,7 +57,7 @@ const splitIprEvidenceLevels = (kbMatches: KbMatchType[]) => {
   return iprRelevanceDict;
 };
 
-const processPotentialClinicalAssociation = (variant: RapidVariantType) => Object.entries(getVariantRelevanceDict(variant.kbMatches))
+const processPotentialClinicalAssociation = (variant: any) => Object.entries(getVariantRelevanceDict(variant.kbMatches))
```

Comment:
uhh ...?

Reply:
srry typescript was throwing some errors I'll try to fix it